banzai_nres/utils/fits_utils.py

```diff
@@ -0,0 +1,28 @@
+import os
+import tempfile
+import shutil
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
```

Comment:
This is a copy (with minor changes) of the image._writeto method from banzai. I would rather update banzai upstream so that image._writeto just called the following writeto function, that way I could import it into banzai_nres. Although I guess that is not such a big deal.